tv_apps/tv-ftu/index.html

```diff
@@ -27,7 +28,7 @@
     <script defer src="js/tv_ftu.js"></script>
 
     <!-- Shared L10n libraries -->
-    <script defer src="shared/js/intl/l20n.js"></script>
+    <script defer src="shared/js/l10n.js"></script>
 
```

Comment:
Since this patch will be uplifted to v2.5 and the stable l20n.js is not in v2.5, so we use l10n.js first.